### DISCUSSION TITLE
How to include page search in search results?

### DISCUSSION THREAD

**Discussion by nicolesherman00** [microsoft-search/pnp-modern-search#2454] [Q&A] [answered]
Hi! I know there is the IsDocument:1 that you can add to your query to only include documents, how could you also include site pages??

**Comment by wobba** [comment, marked as answer]
`PromotedState:0` <- Modern pages
`PromotedState:2` <- News pages

Does this cover it for you?

**Reply by nicolesherman00** [reply]
Yes thank you so much
